modules/moa-installing-prerequisites.adoc

```diff
@@ -0,0 +1,216 @@
+// Module included in the following assemblies:
+//
+// getting_started_moa/creating-first-moa-cluster.adoc
+
+
+[id="moa-installing-prerequisites"]
+= Installing prerequisites
+
+Complete the following prerequisites before creating your {product-title} cluster.
+
+.Procedure
+
+. Select an AWS account to use
```

Comment:
Should each step be a complete sentence? "Select the AWS account you want to use."